Tests/ApplicativeRouterTests/TestRouter.swift

```diff
@@ -4,6 +4,7 @@ import Either
 import Prelude
 
 enum Routes {
+  case home
```

Comment:
we we're seeing this bug in the tests cause the `root` route technically when to the url `/home`. I've now create a separate `home` route so that we can test both and fully root URL and something with path components.